e2e_test/source_inline/cdc/mysql/mysql_create_drop.slt

```diff
@@ -5,6 +5,11 @@ control substitution on
 statement ok
 ALTER SYSTEM SET max_concurrent_creating_streaming_jobs TO 1;
 
+system ok
+mysql -e "
+    SET GLOBAL time_zone = '+01:00';
+"
+
```

Comment:
This will actually affect other potential mysql test running in parallel, do you have any suggestion on addressing this? @xxchan

Reply:
If really needed, you can write the old value to some tmp file, and restore it at the end.